src/components/BrandShoping/BrandShop.jsx

```diff
@@ -0,0 +1,25 @@
+import { React, Fragment } from "react";
+import appleImg from "../../Assets/Images/58687-logo-computer-apple-icons-free-hd-image.png";
+import oneplusImg from "../../Assets/Images/OnePlus-Rebrand-Red.png";
+import samsungImg from "../../Assets/Images/Samsung-Logo.png";
```

Comment:
For static image put them into ```public``` folder  📁 as it will decrease imports in code for assets